harmonica/gravity_corrections.py

```diff
@@ -1,13 +1,18 @@
 """
-Gravity corrections like Normal Gravity and Bouguer corrections.
+Gravity corrections like Bouguer corrections, free air, spherical cap and 
+atmospheric.
+
+Refer to Hinze(2005) for formula.
+http://library.seg.org/doi/10.1190/1.1988183
+
 """
 import numpy as np
 
 from .constants import GRAVITATIONAL_CONST
 
 
 def bouguer_correction(topography, density_crust=2670, density_water=1040):
-    r"""
+    """
```

Comment:
@craigmillernz any reason for this? The `r` is usually needed when writing math to avoid having to use `\\` instead of `\`.

Reply:
opps my mistake, I thought it a typo

Reply:
```suggestion
    r"""
```